src/diagnostics.rs

```diff
@@ -20,7 +20,12 @@ pub fn report_diagnostic<'tcx, 'mir>(
             let info = info.downcast_ref::<TerminationInfo>().expect("invalid MachineStop payload");
             match info {
                 TerminationInfo::Exit(code) => return Some(*code),
-                TerminationInfo::Abort => format!("the evaluated program aborted execution"),
+                TerminationInfo::Abort(opt_msg) =>
+                    if let Some(msg) = opt_msg {
+                        format!("the evaluated program aborted execution: {}", msg)
+                    } else {
+                        format!("the evaluated program aborted execution")
+                    },
```

Comment:
`the evaluated program aborted execution: None` and `the evaluated program aborted execution: Some("attempted...")` looked really weird. so I just conditionally formats it.

Could also be done like: `format!("the evaluated program aborted execution: {}", msg.as_ref().map(|m|format!(": {}", m).unwrap_or_default()))` but I think the simpler if let is more readable even if it duplicates the string

Reply:
> the evaluated program aborted execution: None and the evaluated program aborted execution: Some("attempted...") looked really weird. so I just conditionally formats it.

Yes, debug-printing the `Option` clearly makes no sense. :)